app/models/ncr/work_order.rb

```diff
@@ -6,15 +6,17 @@ def self.table_name_prefix
 
   DATA = YAML.load_file("#{Rails.root}/config/data/ncr.yaml")
 
+  EXPENSE_TYPES = %w(BA61 BA80)
+
+  BUILDING_NUMBERS = DATA['BUILDING_NUMBERS']
+  OFFICES = DATA['OFFICES']
+
   class WorkOrder < ActiveRecord::Base
     # In practice, each work order only has one proposal
     has_many :proposals, as: :client_data
```

Comment:
Proposal has a `client_data_type` and a `client_data_id`, via the `belongs_to` relationship. The `has_many` here is just the other side of that. Looks like I can do `has_one` instead? Does that make sense?